### DISCUSSION TITLE
Version of MEGAN6 used in creation of hemat_transcriptome_v1.6

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1347] [Q&A] [answered]
Hi all,

I'm writing up the draft of my paper, and am having some problems wrapping my head around the steps taken to create hemat_transcriptome_v1.6. I've been going through Sam's notebook posts, but still have some questions (particularly around where the taxonomic filtering takes place).

If someone could write up a quick workflow of the steps taken to create the transcriptome, and the version of MEGAN6 used to do so, that would be extremely helpful! 

Here are some of the notebook posts I've been looking at:
https://robertslab.github.io/sams-notebook/2020/08/14/Transcriptome-Annotation-Hematodinium-Transcriptomes-v1.6-v1.7-v2.1-v3.1-with-DIAMOND-BLASTx-on-Mox.html

https://robertslab.github.io/sams-notebook/2020/08/17/TransDecoder-Hematodinium-Transcriptomes-v1.6-v1.7-v2.1-and-v3.1-on-Mox.html

https://robertslab.github.io/sams-notebook/2021/03/09/Transcriptome-Annotation-Trinotate-Hematodinium-v1.6-on-Mox.html

https://robertslab.github.io/sams-notebook/2020/08/14/Transcriptome-Assessment-BUSCO-Metazoa-on-Hematodinium-v1.6-v1.7-v2.1-and-v3.1-on-Mox.html

**Comment by kubu4** [comment, marked as answer]
See if you can find a crab paper in Pub-a-thon. All info should be in one of them in a format that's all ready to copy and paste!

**Reply by aspencoyle** [reply]
The closest I found was this, from Grace's paper (Chapter 2 of her thesis):

`Sequences were functionally annotated and taxonomically categorized with a combination of DIAMOND BLASTx (0.9.26; (Huson et al. 2016)) and MEGAN6 (6.18.3; (Huson et al. 2016)). Annotation with DIAMOND BLASTx was run against NCBI nr database (downloaded 20190925). The resulting DAA files were converted to RMA6 files for importing into MEGAN6 with the daa2rma utility, using the following MEGAN6 mapping files: prot_acc2tax-Jul2019X1.abin, acc2interpro-Jul2019X.abin, acc2eggnog-Jul2019X.abin. All eukaryotic sequences, sans those categorized within and below Alveolata, were extracted using MEGAN6 to produce the transcriptome assembly used for all analysis.
`

My transcriptome is parasite-specific, and so is definitely not the one she's working with here. Is the overall workflow the same? And is it the same version of MEGAN6?

**Reply by kubu4** [reply]
Yep! The only change would be: `All eukaryotic sequences categorized within and below Alveolata were extracted using MEGAN6 to produce the transcriptome assembly used for all analysis.`
